pkg/sfu/buffer/rtpstats.go

```diff
@@ -1747,3 +1774,90 @@ func AggregateRTPDeltaInfo(deltaInfoList []*RTPDeltaInfo) *RTPDeltaInfo {
 		Firs:                 firs,
 	}
 }
+
+// -------------------------------------------------------------------
```

Comment:
putting the implementation in this file for now. If this works, will clean up and move it to a separate module.